### PR TITLE
HTML + CSS Mastery Exercise

### PR DESCRIPTION
**Pull request by stevenwally**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Steven Wallace Holmes</title>
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+	<!-- <link rel="stylesheet" href="animation.css"> -->
```
Comment (gregkorte) [inline index.html]: Try to purge commented code when you turn your projects in.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+		</article>
+
+		<article id="links">
+			<h1>resources</h1>
+			<ul>
+				<li><a href="http://www.codepen.io">CodePen.io</a></li>
+				<li><a href="https://dribbble.com/">Dribbble</a></li>
+			</ul>
+		</article>
+</div>
```
Comment (gregkorte) [inline index.html]: Watch your indentation.

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+#bio h1,
+#projects h1 {
+	font-family: 'Cutive', serif;
+	display: block;
+	color: rgba(66, 55, 55, 0.60);
+	line-height: .35em;
+	padding-left: 1em;
+	padding-bottom: 1em;
+}
+
```
Comment (gregkorte) [inline css/styles.css:70]: Try to keep consistent spacing in you files. It makes for easier reading.

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+#wallace {
+	letter-spacing: 7.5px;
+}
+
+#holmes {
+	letter-spacing: 10px;
+}
+
+/*#titles-div h1 {
```
Comment (gregkorte) [inline css/styles.css]: Try to purge commented code when you turn your projects in.

**Comment by gregkorte** [comment]
Nice work on your quiz, Steven! See the few comments I made and make your changes. Once you're finished you are good to merge. :+1:

**Comment by gregkorte** [comment]
Sorry, didn't mean to close your PR.

**Comment by stevenwally** [comment]
thanks for the comments, Greg!

**Pull request merged**